packages/cli/src/commands/deploy/index.ts

```diff
@@ -194,6 +201,28 @@ export default async (client: Client) => {
       );
       return 1;
     }
+
+    const prebuiltBuild = await fs.readJSON(
+      join(path, '.vercel/output/builds.json'),
+      { throws: false }
+    );
+    const assumedTarget = target || 'preview';
```

Comment:
This defaults the target to "preview" for the purposes of comparison only. If we did this at the root of the CLI, other things will break.